track/log.py

```diff
@@ -0,0 +1,107 @@
+"""
+File adopted from Michael Whittaker.
+
+This module firstly offers a convenience function called "debug" which
+alleviates a couple of inconveniences in python logging:
+
+* No need to find a logger before logging (uses the one from this package)
+* Slightly friendly string interpolation interface.
+"""
+
+from datetime import datetime
+import inspect
+import hashlib
+import subprocess
+import os
+import shlex
+import json
+import sys
+import logging
+
+class TrackLogHandler(logging.FileHandler):
+    """File-based logging handler for the track package"""
+    pass
+
+class StdoutHandler(logging.StreamHandler):
+    """As described by the name"""
+    def __init__(self):
+        super().__init__(sys.stdout)
+
+def init(track_log_handler):
+    """
+    (Re)initialize track's file handler for track package logger.
```

Comment:
what exactly is this doing? 

Is the idea that users will log things via

Reply:
No, they should log using track.debug()